docs/whats-new/dotnet-docs-mod2.md

```diff
@@ -1,95 +1,132 @@
+- [SYSLIB0046: ControlledExecution.Run should not be used](../fundamentals/syslib-diagnostics/syslib0046.md)
+- [CA2019: `ThreadStatic` fields should not use inline initialization](../fundamentals/code-analysis/quality-rules/ca2019.md)
+- [CA2259: Ensure `ThreadStatic` is only used with static fields](../fundamentals/code-analysis/quality-rules/ca2259.md)
+- [CA1420: Property, type, or attribute requires runtime marshalling](../fundamentals/code-analysis/quality-rules/ca1420.md)
+- [CA1422: Validate platform compatibility - obsoleted APIs](../fundamentals/code-analysis/quality-rules/ca1422.md)
+- [System.diagnostics entry in app.config](../core/compatibility/configuration/7.0/diagnostics-config-section.md)
+- [WebSockets support in .NET](../fundamentals/networking/websockets.md)
+- [API obsoletions with default diagnostic ID (.NET 7)](../core/compatibility/core-libraries/7.0/obsolete-apis-with-default-diagnostic.md)
+- [System.Runtime.CompilerServices.Unsafe NuGet package](../core/compatibility/core-libraries/7.0/unsafe-package.md)
+- [Environment variable prefixes](../core/compatibility/extensions/7.0/environment-variable-prefix.md)
+- [Legacy FileStream strategy removed](../core/compatibility/core-libraries/7.0/filestream-compat-switch.md)
+- [Time fields on symbolic links](../core/compatibility/core-libraries/7.0/symbolic-link-timestamps.md)
 
 ### Updated articles
 
-- [Composite formatting](../standard/base-types/composite-formatting.md)
-- [Install .NET on Windows](../core/install/windows.md) - Add winget install / Update linux tables
-- [Quantifiers in Regular Expressions](../standard/base-types/quantifiers-in-regular-expressions.md) - [Freshness] US-1953530: Refresh top articles by PV - .NET and Languages Content (4)
+- [dotnet-coverage code coverage utility](../core/additional-tools/dotnet-coverage.md) - Add static instrumentation option into documentation
+- [Install the .NET SDK or the .NET Runtime on Ubuntu](../core/install/linux-ubuntu.md) - Updates for linux install instructions
+- [TCP overview](../fundamentals/networking/sockets/tcp-classes.md) - Equivalent usages section in TCP
 
-## Architecture guides
+## C# language
 
 ### New articles
 
-- [Enterprise Application Patterns Using .NET MAUI](../architecture/maui/index.md): new Architecture eBook
+- [Resolve warnings related to language features and versions](../csharp/language-reference/compiler-messages/feature-version-errors.md)
+- [Pattern matching warnings](../csharp/language-reference/compiler-messages/pattern-matching-warnings.md)
+
+## Microsoft Orleans
 
-## Azure SDK for .NET
+### New articles
+
+- [What's new in Orleans 7.0](../orleans/whats-new-in-orleans.md)
 
 ### Updated articles
 
-- [Resource management using the Azure SDK for .NET](../azure/sdk/resource-management.md) - update resource management docs keep align with latest SDK
+- [What's new in Orleans 7.0](../orleans/whats-new-in-orleans.md)
```

Comment:
I'd just remove the updated articles section completely for Orleans.

Reply:
Can the tool add a check to not add an article as "updated" if it is also "new"?